c7n/mu.py

```diff
@@ -807,7 +814,7 @@ def memory_size(self):
 
     @property
     def timeout(self):
-        return self.policy.data['mode'].get('timeout', 60)
+        return self.policy.data['mode'].get('timeout', 900)
```

Comment:
Wait, that's a timeout of 15 minutes (60*15 = 900). Don't we want a timeout instead of 5 minutes, which is 5 min (60*5 = 300) seconds?

Reply:
15 * 60 == 900 .. I figured just set it to the max possible, to avoid having to configure it unless folks are specifically trying to decrease the timeout (specialized use case).

Reply:
@kapilt ok that's fair.